hknweb/alumni/migrations/0004_auto_20191003_2323.py

```diff
@@ -0,0 +1,18 @@
+        ),
+    ]
```

Comment:
Somehow this migration file causes the migrate to brea, due to multiple migrations having the 0004 name. Please fix this before merging.